src/components/_templates/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,81 @@
+import { Button, Checkbox, Form, Input, Space, Typography } from 'antd'
+import { LockOutlined, MailOutlined } from '@ant-design/icons'
+import { useNavigate } from 'react-router-dom'
+
+import { requestSignIn } from '../../../redux/users/action'
+import { useAppDispatch } from '../../../redux/hooks'
+import { RULES_FORM } from '../../../helper/helper'
+import { SignInData } from './api'
+import '../SignIn/style.scss'
+
+type FormData = {
+  username: string
+  password: string
+}
+
+const prepareUser = (formData: FormData): SignInData => ({
```

Comment:
Does it necessary to use it? It used only one time. But if u want to use it, u should put it to "utils" file